static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -453,6 +453,11 @@ const SPECIAL_FUNCTIONS: SpecialFunctions = {
       return <NumberContainer>{emptyValue}</NumberContainer>;
     }
 
+    const userMisery = data[userMiseryField];
+    if (userMisery === null || isNaN(userMisery)) {
```

Comment:
I've started using `defined()` from `app/utils` everywhere since we've got edge cases from solely checking for just undefined or just null.

Reply:
oh nice, I'll start using it too. just going to merge this as is to get it into prod